src/main/kotlin/gg/essential/vigilance/gui/VigilancePalette.kt

```diff
@@ -50,6 +50,7 @@ object VigilancePalette {
     fun getScrollbar(): Color = scrollbar.get()
     fun getTextShadow(): Color = textShadow.get()
     fun getMainBackground(): Color = mainBackground.get()
+    fun getEssentialAccentBlue(): Color = essentialAccentBlue.get()
```

Comment:
Should definitely not call it "blue" in the api, that's effectively exposing an implementation detail. Probably shouldn't even say that it's "Essential".
Now, at that point it clashes with an existing method, so how about going with its usage instead and calling it `textActive` to fit with the other text colors?